src/components/boardComponents/Cylinder.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { darken } from "color2k";
+
+const Cylinder = ({ size = 50, color = "#b3b3b3" }) => {
+    const colorDarken1 = darken(color, 0.25);
+
+    return (
+      <svg version="1.1" id="fi_758454" xmlns="http://www.w3.org/2000/svg" height={size} width={size} viewBox="0 0 377.208 377.208">
+        <path style={{fill: colorDarken1}} d="M188.604,111.804c-85.159,0-154.645-22.988-159.869-52.245h-0.522v261.747
```

Comment:
I think the most visible part should have the main color and you can use the ligthen one for the top, what do you think ?